src/cargo.md

```diff
@@ -14,5 +14,17 @@ $ sudo apt install cargo rust-src
 This will allow [rust-analyzer][1] to jump to the definitions. We suggest using
 [VS Code][2] to edit the code (but any LSP compatible editor works).
 
+As a sidenote: if you have the access/capability to do so, it's recommended to
+install Rust's tooling via [rustup](https://rustup.rs/) since it's better integrated with the
+rest of the ecosystem. As of writing, the way to install `rustup` on Unix system would be
+
+```shell
+$ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
Please drop this part and just refer people to rustup.rs — that simplifies our job and then there can be one canonical place in the ecosystem where this magic string is maintained.

Reply:
sure